packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -36,8 +35,10 @@ import {
   blobToFileObj,
   contentKeyFromBranch,
   generateContentKey,
+  localForage,
+  allEntriesByFolder,
 } from 'netlify-cms-lib-util';
-import NetlifyAuthenticator from 'netlify-cms-lib-auth';
+import { NetlifyAuthenticator } from 'netlify-cms-lib-auth';
```

Comment:
This fixes https://github.com/netlify/netlify-cms/issues/3288, there is code to handle Bitbucket token expiration, it just wasn't importing the class correctly.